workflow/controller/controller.go

```diff
@@ -535,19 +535,63 @@ func (wfc *WorkflowController) processNextPodCleanupItem(ctx context.Context) bo
 				return err
 			}
 		case labelPodCompleted:
-			_, err := pods.Patch(
+			patch := []common.DeleteFinalizerPatch{
+				{
+					Operation: "replace",
+					Path:      "/metadata/labels/workflows.argoproj.io~1completed",
+					Value:     "true",
+				},
```

Comment:
is this label right? It's not supposed to be `"workflows.argoproj.io/completed"`, is it?

Reply:
if so, you can use `common.LabelKeyCompleted`

Reply:
`~1` is escape for JSON Pointer.
https://fdatatracker.ietf.org/doc/html/rfc6901#section-3

Reply:
okay, thanks